starters/basic-starter/pages/[...slug].tsx

```diff
@@ -111,32 +137,130 @@ export async function getStaticProps(
     params.addInclude(['field_place_image.image']);
   }
 
-  // Fetch the node/resource from Drupal.
-  const node = await drupal.getResourceFromContext<DrupalNode>(type, context, {
-    params: params.getQueryObject(),
-  });
-
-  // At this point, we know the path exists and it points to a resource. If we
-  // receive an error, it means something went wrong on Drupal. We throw an
-  // error to tell revalidation to skip this for now. Revalidation can try again
-  // on next request.
-  if (!node) {
-    throw new Error(`Failed to fetch resource: ${path.jsonapi.individual}`);
+  let node_list;
+  let node;
+  // Fetch the resource from Drupal.
+  switch (type) {
+    case 'taxonomy_term--person_type': {
+      node_list = await drupal.getResourceCollectionFromContext<DrupalNode[]>(
+        'node--person',
+        context,
+        {
+          params: new DrupalJsonApiParams()
+            .addInclude(['field_person_image.image', 'field_person_type'])
+            .addFilter('field_person_type.name', label)
+            .getQueryObject(),
+        },
+      );
+      break;
+    }
+    case 'taxonomy_term--article_type': {
+      node_list = await drupal.getResourceCollectionFromContext<DrupalNode[]>(
+        'node--article',
+        context,
+        {
+          params: new DrupalJsonApiParams()
+            .addInclude([
```

Comment:
We should probably sort all of the lists. Maybe sorting by the creation time would make sense (this would be same as what we do in Drupal)?